src/codemodder/__main__.py

```diff
@@ -192,6 +192,14 @@ def run(argv, original_args) -> int:
     return 0
 
 
-if __name__ == "__main__":
+def run_script():
```

Comment:
what about calling it `main`? don't love the wording with script

Reply:
Yeah that's a better idea.